poetry.lock

```diff
@@ -1708,14 +1708,14 @@ files = [
 
 [[package]]
 name = "types-protobuf"
-version = "3.20.3"
+version = "4.21.0.6"
```

Comment:
I wasn't sure we should also do the upgrade here, but seeing as how we do it for `protobuf` in the lock file this makes plenty of sense

Reply:
I believe it doesn't really matter, this lockfile should only affect you as a library developer and not the end users.
Also in its current state it's incosistent, since protobuf types don't match protobuf itself, which is I guess incorrect.
So I don't see a reason to not doing it.

Reply:
True! Agreed.